store/storetest/webhook_store.go

```diff
@@ -43,13 +43,11 @@ func TestWebhookStore(t *testing.T, ss store.Store) {
 func testWebhookStoreSaveIncoming(t *testing.T, ss store.Store) {
 	o1 := buildIncomingWebhook()
 
-	if _, err := ss.Webhook().SaveIncoming(o1); err != nil {
-		t.Fatal("couldn't save item", err)
-	}
+	_, err := ss.Webhook().SaveIncoming(o1)
+	require.Nil(t, err, "couldn't save item")
 
-	if _, err := ss.Webhook().SaveIncoming(o1); err == nil {
-		t.Fatal("shouldn't be able to update from save")
-	}
+	_, err = ss.Webhook().SaveIncoming(o1)
+	require.NotNil(t, err, "shouldn't be able to update from save")
```

Comment:
There are `Error`/`NoError` functions in `require` package
- https://godoc.org/github.com/stretchr/testify/require#Error
- https://godoc.org/github.com/stretchr/testify/require#NoError

Reply:
You are right, but we are using a custom error type and the previous version of testify wasn't working well with custom errors (at least in our case) so for consistency is ok to keep is as `Nil`/`NotNil` for now, and change it to `Error`/`NoError` later.